src/server/pfs/server/trigger.go

```diff
@@ -34,8 +34,8 @@ func (d *driver) triggerCommit(
 	}
 	// find which branches this commit is the head of
 	headBranches := make(map[string]bool)
-	bi := &pfs.BranchInfo{}
 	for _, b := range repoInfo.Branches {
+		bi := &pfs.BranchInfo{}
```

Comment:
Declaring bi here fixes an issue that was discovered while writing the docs that caused pachd to segfault.